tyk/api/request.go

```diff
@@ -1,9 +1,17 @@
 package api
 
+type EventConfig struct {
+	Webhook string `json:"webhook,omitempty"`
+	Email   string `json:"email,omitempty"`
+	Redis   bool   `json:"redis,omitempty"`
```

Comment:
I don't see reference to this field in the docs https://tyk.io/docs/tyk-multi-data-centre/setup-controller-data-centre/#gateway-configuration